lib/mixins/bindings.js

```diff
@@ -0,0 +1,46 @@
+const { getMapValueByKey } = require('../utils');
+
+/**
+ * Implements functions to deal with the common Bindings object.
+ * @mixin MixinBindings
+ */
+const MixinBindings = {
+  /**
+   * @returns {boolean}
+   */
+  hasBindings() {
+    return !!(this._json.bindings && Object.keys(this._json.bindings).length);
```

Comment:
I don't know that return true only if bindings, externalDocs etc. have not only a definition, but also properties/items. It needs to be discussed.